demos/core-us-investment-style/demo.js

```diff
@@ -133,8 +133,10 @@ async function displayXRayUS (postmanJSON) {
             name: 'Portfolio Weight',
             borderWidth: 1,
             borderColor: '#e5e7e9',
-            keys: ['Type', 'value', '_', 'x', 'y'],
-            data: connector.dataTables.EquityStyle.getRows(0, 9),
+            keys: ['Type', 'value', 'x', 'y'],
+            data: connector.dataTables.EquityStyle.getRows(
+                void 0, void 0, ['Type', 'Value', 'Style', 'Size']
+            ),
```

Comment:
Cool! But does it skip 10th category ("Unclassified")? Bottom left corner probably has two points

Reply:
I have double checked everything, but it seems to be absolutely fine 🤔 
Here is the output that we use to populate the stylebox, I have also compared it with original dataTable values and they are matching.

<img width="328" height="232" alt="image" src="https://github.com/user-attachments/assets/90ab6120-64b1-423a-a20c-473414fa76b4" />

Reply:
Okay, double-checked. It is fine, Unclassified has a separate column (XRay EU populates this here, category 99:
<img width="832" height="204" alt="image" src="https://github.com/user-attachments/assets/afefdb66-6174-41f2-af86-7df703ee2c02" />
).